varats/data/reports/taint_report.py

```diff
@@ -0,0 +1,49 @@
+"""
+Module for all reports generated for taint flow analyses."
+"""
+
+import os
+from pathlib import Path
+
+from varats.data.report import BaseReport, FileStatusExtension
+
+
+class TaintPropagationReport(BaseReport):
+    """
+    Print the result of filechecking a llvm ir file generated by VaRA or Phasar
+    in a readable manner.
+    """
+
+    SHORTHAND = "TAINT"
+    FILE_TYPE = "txt"
+
+    def __init__(self, path: Path) -> None:
+        super(TaintPropagationReport, self).__init__()
+        with open(path, "r") as stream:
+                self._path = path
```

Comment:
```suggestion
                self.__path = path
```
private vars should have two underscores